test/e2e/tests/dapp-interactions/block-explorer.spec.js

```diff
@@ -85,6 +85,7 @@ describe('Block Explorer', function () {
         const [, tst] = await driver.findElements(
           '[data-testid="multichain-token-list-button"]',
         );
+
```

Comment:
We can remove this to make the PR smaller.

Reply:
Removed 🗑️